src/game.zig

```diff
@@ -486,8 +486,8 @@ pub fn flyToggle() void {
 }
 
 pub fn update(deltaTime: f64) void {
-	const gravity = 30.0;
-	const terminalVelocity = 90.0;
+	const gravity = 32.0;
+	const terminalVelocity = 128.0;
```

Comment:
Could you explain the reason for changing the terminal velocity of flying?

Reply:
Nah, it's the velocity you reach when falling down under only the influence of gravity.

Reply:
Maximum velocity is unlimited.